src/api/schema/metrics/mod.rs

```diff
@@ -108,30 +108,30 @@ impl MetricsSubscription {
     }
 
     /// Component events processed metrics over `interval`.
-    async fn component_events_processed_totals(
+    async fn component_processed_events_totals(
         &self,
         #[graphql(default = 1000, validator(IntRange(min = "10", max = "60_000")))] interval: i32,
-    ) -> impl Stream<Item = Vec<ComponentEventsProcessedTotal>> {
-        component_counter_metrics(interval, &|m| m.name == "events_processed_total").map(|m| {
+    ) -> impl Stream<Item = Vec<ComponentProcessedEventsTotal>> {
+        component_counter_metrics(interval, &|m| m.name == "processed_events_total").map(|m| {
             m.into_iter()
-                .map(ComponentEventsProcessedTotal::new)
+                .map(ComponentProcessedEventsTotal::new)
                 .collect()
         })
     }
 
     /// Bytes processed metrics.
-    async fn bytes_processed_total(
+    async fn processed_bytes_total(
         &self,
         #[graphql(default = 1000, validator(IntRange(min = "10", max = "60_000")))] interval: i32,
-    ) -> impl Stream<Item = BytesProcessedTotal> {
+    ) -> impl Stream<Item = ProcessedBytesTotal> {
         get_metrics(interval).filter_map(|m| match m.name.as_str() {
-            "processed_bytes_total" => Some(BytesProcessedTotal::new(m)),
+            "processed_bytes_total" => Some(ProcessedBytesTotal::new(m)),
             _ => None,
         })
     }
 
     /// Bytes processed throughput, sampled over a provided millisecond `interval`.
```

Comment:
I might also change the comments to reflect "Processed bytes" vs. "Bytes processed"